docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -59,6 +119,50 @@ const compactHumanizeDate = (date) => {
     .replace(" year", "y");
 };
 
+/**
+ * Convert an Optional string of high, medium, low to a number of 0, 1, 2, 3
+ */
+const levelToIcon = (level) => {
+  switch (level?.toLowerCase()) {
```

Comment:
See my [comment on the other PR](https://github.com/airbytehq/airbyte/pull/38799#discussion_r1628482487) about how the syncSuccessRate value is a boolean on some connectors (e.g. ActiveCampaign), which causes this error when trying to visit those connectors on this PR:
<img width="1690" alt="Screenshot 2024-06-05 at 3 31 27 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/ec280094-a7a2-4d3d-8846-05ae12261c8b">

Reply:
Corrected this in the previous PR. And removed this function in this pr